stable/provazio/templates/dashboard-service.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.dashboard.enable }}
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "provazio.dashboard.name" . }}
+  labels:
+    app: {{ template "provazio.dashboard.name" . }}
+    chart: {{ template "provazio.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  ports:
+  - name: admin
+    port: 8060
+    protocol: TCP
+    targetPort: 8060
+  selector:
+    provazio/service: {{ template "provazio.dashboard.name" . }}
```

Comment:
Selector should include all the labels used by Helm

Reply:
Done